README.md

```diff
@@ -89,40 +82,84 @@ df
 |  5 |           4 |        1 |      3 |            0 |           1 |
 ```
 
-You can try out Raphtory for yourself in a Jupyter Notebook. Please click here [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Raphtory/Raphtory/v0.2.0a9?labpath=examples%2Fbinder_python%2Findex.ipynb) to launch the notebook.
-
 # Installing Raphtory 
-Raphtory is available for both Python and Scala / Java, with support for Rust planned in the 0.3.0 release. 
 
-We recommend installing the PyRaphtory client for Python, which contains all the functions to create a graph, run algorithms and analyse results. We also support Conda, but pip is the preferred option.
+Raphtory is available for Python and Scala/Java, with support for Rust planned in version 0.3.0. We recommend using the PyRaphtory client for Python, which includes everything you need and can be run locally or in distributed mode.
 
-Make sure you have **Python v3.9 or above**. It is also recommended that you install Raphtory through conda, virtualenv or pyenv as a best practice. 
+You should have Python version 3.9 or higher. It's a good idea to use conda, virtualenv, or pyenv. 
 
 ```bash
 pip install pyraphtory_jvm pyraphtory
-```
+``` 
+
+# Examples and Notebooks
+
+Check out Raphtory in action with our interactive Jupyter Notebook! Just click the badge below to launch a Raphtory sandbox online, no installation needed.
+
+ [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Raphtory/Raphtory/v0.2.0a9?labpath=examples%2Fbinder_python%2Findex.ipynb) 
```

Comment:
if u merge with master than this points to master branch instead of v0.2.0a9 

is there a way to make it point to the latest release so we dont have to always update it?

Reply:
I will check if there is a badge or tool. Otherwise we would need latest tag

Reply:
Sadly not. We would need to create a tag called latest instead. Which is not great (conflicts with github symmantics).